CDCgov/prime-reportstream
# 7403 remove header within Resources and Support child pages

Test Steps:
1. Go to [resources](https://reportstream.cdc.gov/resources) and [support]()
2. See section header on index page (ex: "Resources -- Explore guides, tools, and resources to optimize ReportStream")
2. Click through to any show page (ex: https://reportstream.cdc.gov/resources/account-registration-guide or https://reportstream.cdc.gov/support/faq)
3. Do not see section header within child page

## Changes
- Removed header within Resources and Support child pages
<img width="1767" alt="Screenshot 2022-11-21 at 8 32 47 PM" src="https://user-images.githubusercontent.com/102491809/203222287-b9c7fa56-5e44-46cc-b7ce-1e483c33cc42.png">
<img width="1756" alt="Screenshot 2022-11-21 at 8 32 54 PM" src="https://user-images.githubusercontent.com/102491809/203222293-19f4377b-cadb-458e-b417-b813a3b5c65b.png">
<img width="1782" alt="Screenshot 2022-11-21 at 8 33 00 PM" src="https://user-images.githubusercontent.com/102491809/203222297-3090fb91-5239-4796-a676-067a47f91a1e.png">
<img width="1775" alt="Screenshot 2022-11-21 at 8 33 07 PM" src="https://user-images.githubusercontent.com/102491809/203222299-ca95aecc-8bc0-4b79-889e-eeba32d49b4f.png">

## Checklist

### Testing
